.github/workflows/publish_public_package.yaml

```diff
@@ -0,0 +1,105 @@
+# Purpose: Publish on demand to the real gallery (PSGallery).
+
+name: Publish Public Package
+
+on:
+  workflow_dispatch:
+    inputs:
+      OverrideModuleVersion:
+        description: "Override the version of the release. Restricted to SemVer 1.0 - 3 segments"
+        required: false
+        type: string
+      IsPrerelease:
+        description: "Is this a prerelease"
+        required: false
+        type: boolean
+        default: false
+      PrereleaseTag:
+        description: "The prerelease tag: [0-9A-Za-z]+"
+        required: false
+        type: string
+  push:
+    paths:
+      - ".github/workflows/publish_public_package.yaml"
+      - "utils/DeployUtils.ps1"
```

Comment:
They are probably helpful, just in case someone else edits DeployUtils, but I commented them out and added a comment that they were helpful for debugging.